### ISSUE TITLE
execution fails due to inaccessibility of GitHub API

### ISSUE THREAD

**Issue by yegor256** [zerocracy/judges-action#314]
Happened here: https://github.com/objectionary/eo/actions/runs/10534788751/job/29192922574

I'm not sure what could be a solution though...
